### PR TITLE
Create ccpp.yml

### PR DESCRIPTION
**Pull request by Mab168**

### CODE REVIEW

File: .github/workflows/ccpp.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    steps:
+    - uses: actions/checkout@v2
+    - name: configure
+      run: ./configure
+    - name: make
+      run: make
+    - name: make check
+      run: make check
+    - name: make distcheck
+      run: make distcheck
```
Comment (Mab168) [inline .github/workflows/ccpp.yml:19]: Applications

Comment (Mab168) [reply .github/workflows/ccpp.yml:19]: M A B-logo

Comment (Mab168) [reply .github/workflows/ccpp.yml:19]: Founder-icon

**Review by Mab168 — COMMENTED** [review]
Approved

**Comment by luke-jr** [comment]
What does this do?

**Pull request closed without merging**